api/weather/types.go

```diff
@@ -58,6 +58,6 @@ type OpenWeatherResponse struct {
 	Sys        sys
 	Timezone   float64
 	ID         float64
-	Name       string
+	Name       string `json:"name"`
```

Comment:
this shouldn't be necessary as the JSON unmarshalling knows to go from lower to uppercase?

Reply:
Yeah I didn't know that at the time of writing. I can go through and remove those. Although the `json:"-"` is still needed to ignore those fields.